### ISSUE TITLE
Really long paths cause left-aligned second line wrapping in admin interface

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#502]
![screen shot 2018-08-14 at 5 27 53 pm](https://user-images.githubusercontent.com/2514780/44119516-7c11b0e6-9fe7-11e8-98da-65f7fe055446.png)

This is testing with docker, so the paths are a bit longer but really any long path will cause this. Note the index path wrapping.

**Comment by machawk1** [comment]
Perhaps it should be a column-based display where the second line wraps to the second column. As an alternative, we could also use "..." or some other UI overflow mechanism.

**Comment by ibnesayeed** [comment]
CSS Grid can solve this issue very easily. And yes, ellipsis is another good alternative, but we cannot rely on automatic ellipsis behavior as it will clip the text from the end which is more important than the long path prefixes.

**Comment by machawk1** [comment]
Ultimately this information will not be restricted to its small width (once #213 is complete), so in the meantime we ought to get it at least wrapping a little more aesthetically.

**Comment by machawk1** [comment]
I am unable to replicate this display with `ipwb replay ipwb/samples/indexes/sample-1.cdxj` but the misalignment likely still exists.
